spec/system/support_interface/feature_metrics_dashboard_spec.rb

```diff
@@ -56,6 +57,34 @@ def reject_application(application_choice)
     )
   end
 
+  def apply_again_and_reject_application(application_form)
+    apply_again_application_form = DuplicateApplication.new(
+      application_form,
+      target_phase: 'apply_2',
+    ).duplicate
+    application_choice = create(
+      :application_choice,
+      :awaiting_provider_decision,
+      application_form: apply_again_application_form,
+    )
+    ApplicationStateChange.new(application_choice).make_offer!
+    apply_again_application_form
+  end
+
+  def apply_again_and_offer_application(application_form)
```

Comment:
i think these two methods are the wrong way round. This one rejects the application and the other one makes an offer.